src/lib.rs

```diff
@@ -51,6 +52,8 @@ impl From<core::str::Utf8Error> for ScrollerError {
         ScrollerError
     }
 }
+
+#[cfg(not(version("1.61")))]
```

Comment:
This one causes problems?

Reply:
yeah.. `core::option::NoneError` is now gone